docs/api/index.md

```diff
@@ -78,22 +78,21 @@ app.listen(3000);
 const Koa = require('koa');
 const app = new Koa();
 
-// x-response-time
+// logger
 
 app.use(async (ctx, next) => {
-  const start = Date.now();
   await next();
-  const ms = Date.now() - start;
-  ctx.set('X-Response-Time', `${ms}ms`);
+  const rt = ctx.response.get('X-Response-Time');
+  console.log(`${ctx.method} ${ctx.url} - ${rt}`);
```

Comment:
it's not good here because `rt` is a string

Reply:
@dead-horse I'm not sure what you mean; can you elaborate please?  Thanks!

Reply:
never mind, my misunderstanding.